### ISSUE TITLE
conda install may have broken (possibly with a recent update?)

### ISSUE THREAD

**Issue by JohnUrban** [marbl/verkko#280]
Hi,

Thanks for the tools and for your time, as usual.

I have been trying to install verkko via conda, using the recommended command:

```
conda create -n verkko -c conda-forge -c bioconda -c defaults verkko
```

or its mamba variant:
```
mamba create -n verkko -c conda-forge -c bioconda -c defaults verkko
```

Both seemed to hang forever.

So I tried the simpler command (https://anaconda.org/bioconda/verkko) and its mamba variant:

```
conda create -n verkko bioconda::verkko
```

or 

```
mamba create -n verkko bioconda::verkko
```

Conda output was:

```
Collecting package metadata (current_repodata.json): done
Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): done
Solving environment: \ 
Found conflicts! Looking for incompatible packages.
This can take several minutes.  Press CTRL-C to abort.
failed                                                                                                                                                                                                                                                                  

UnsatisfiableError: The following specifications were found to be incompatible with your system:

  - feature:/linux-64::__glibc==2.34=0
  - bioconda::verkko -> libgcc-ng[version='>=10.3.0'] -> __glibc[version='>=2.17']

Your installed version is: 2.34
```

Mamba had similar output:
```
Looking for: ['bioconda::verkko']

bioconda/linux-64        Using cache
pkgs/main/linux-64       [====================] (00m:00s) No change
pkgs/main/noarch         [====================] (00m:00s) No change
pkgs/r/linux-64          [====================] (00m:00s) No change
pkgs/r/noarch            [====================] (00m:00s) No change
bioconda/noarch          [====================] (00m:00s) No change
Encountered problems while solving:
  - nothing provides libgcc-ng >=12 needed by verkko-1.1-h64afbab_0
```

Some info about the OS:
```
uname -a
Linux login4.cm.cluster 5.14.0-362.24.1.el9_3.x86_64 #1 SMP PREEMPT_DYNAMIC Thu Feb 15 07:18:13 EST 2024 x86_64 x86_64 x86_64 GNU/Linux
```

I also tried on two other computers.

A mac desktop:
```
uname -a 
Darwin JohnUrbanSpradling.local 22.6.0 Darwin Kernel Version 22.6.0: Mon Jun 24 01:25:37 PDT 2024; root:xnu-8796.141.3.706.2~1/RELEASE_X86_64 x86_64

conda create -n verkko bioconda::verkko
Collecting package metadata (current_repodata.json): done
Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): done
Solving environment: \ 
Found conflicts! Looking for incompatible packages.
This can take several minutes.  Press CTRL-C to abort.
failed                                                                                                                                                                                                                                                                  

UnsatisfiableError: The following specifications were found to be incompatible with your system:

  - feature:/osx-64::__osx==10.16=0
  - bioconda::verkko -> mashmap[version='>=3.0.6'] -> __osx[version='>=10.9']

Your installed version is: 10.16
```


And on another remote linux machine where it is still hanging after 5-10 minutes.

Well, that is the report. Hopefully there is hope for a poor schlub like me.

Maybe it is just a simple fix of requirements to avoid conflicts...?

Thanks again.

Best,

John

**Comment by JohnUrban** [comment]
Could mashmap dependencies be involved?


If I try to install the dependencies:
```
mamba create -n verkko_deps -c bioconda rust python=3.10 snakemake=7.0 mashmap=3.1.3 winnowmap bwa minimap2 samtools seqtk
```

I get this:
```
Looking for: ['rust', 'python=3.10', 'snakemake=7.0', 'mashmap=3.1.3', 'winnowmap', 'bwa', 'minimap2', 'samtools', 'seqtk']

bioconda/linux-64        Using cache
bioconda/noarch          Using cache
pkgs/main/linux-64       [====================] (00m:00s) No change
pkgs/main/noarch         [====================] (00m:00s) No change
pkgs/r/linux-64          [====================] (00m:00s) No change
pkgs/r/noarch            [====================] (00m:00s) No change
Encountered problems while solving:
  - nothing provides libgcc-ng >=12 needed by mashmap-3.1.3-h07ea13f_0
  - nothing provides python-irodsclient needed by snakemake-7.0.0-hdfd78af_0
```

**Comment by skoren** [comment]
So this is resolved? I did have to bump the python version in the v2.2 release due to some new features verkko is using. None of the other versions shouldn't have changed though there were a couple of more dependencies added (pysam and htslib: https://github.com/bioconda/bioconda-recipes/pull/50475/files#diff-f4b4488ab37a20751b1d58e903e1d536e7c765df46b619aecd87527cf5791a5f) but I wouldn't have expected that to break things.

**Comment by JohnUrban** [comment]
Sorry. Yes - I should have closed this. It has installed, and I can def say I get the verkko help message. I am waiting for it to run on a SLURM system now. 

Ultimately... what I think happened is this:
- I tried installing via the conda command
- The connection to the remote system broke
- That left some kind of "lock" file in place, which I guess was the problem....
- Once I erased it, the conda command worked with no problems.
